clusterloader2/testing/load/config.yaml

```diff
@@ -373,7 +378,9 @@ steps:
     Instances:
      - Identifier: WaitForRunningDeployments
      - Identifier: WaitForRunningStatefulSets
+{{if not $DISABLE_DAEMONSETS}}
```

Comment:
is that needed? If there isn't any DS, it shouldn't matter at all, right?

Reply:
Yes, we could skip this part. I personally don't see big difference between having it or not.

Reply:
If you prefer to remove it, let me know and I will do it.

Reply:
The simpler the config is the better.
So if it works without this if too, I prefer removing it.

Reply:
Let me double check to make sure it works without it